lib/core/src/model.rs

```diff
@@ -32,13 +32,26 @@ pub const LIQUID_FEE_RATE_SAT_PER_VBYTE: f64 = 0.1;
 pub const LIQUID_FEE_RATE_MSAT_PER_VBYTE: f32 = (LIQUID_FEE_RATE_SAT_PER_VBYTE * 1000.0) as f32;
 pub const BREEZ_SYNC_SERVICE_URL: &str = "https://datasync.breez.technology";
 
+#[derive(Clone, Debug, Serialize)]
+pub enum BlockchainExplorer {
+    Electrum {
+        url: String,
+    },
+    Esplora {
+        url: String,
+        /// Whether or not to use the "waterfalls" extension
+        use_waterfalls: bool,
+    },
+    MempoolSpace {
```

Comment:
The MempoolSpace is only used to fetch recommended fees and not really used as fully blockchain explorer. I think we could treat it like before by having the mempoolspace_url in the config.
I think it will simplify a lot (and also be more clear) if we have only two variants here:
Electrum
Waterfall

Reply:
I think we should have another look at dropping the mempool.space dependency. Esplora directly provides sat/vb fee rate estimates ([docs](https://github.com/blockstream/esplora/blob/master/API.md#get-fee-estimates)). Electrum provides fees, not fee rates, but we could figure out which tx size they are using for the estimations and convert them to a fee rate ([docs](https://electrum.readthedocs.io/en/latest/protocol.html#blockchain-estimatefee)).